tests/BiosTests.hs

```diff
@@ -31,21 +32,25 @@ main = do
                   (Just "./tests/projects/simple-cabal/hie.yaml")
                 )
         ]
-      , testGroup "Loading tests" [
-        testCaseSteps "simple-cabal" $ testDirectory "./tests/projects/simple-cabal/B.hs"
-        , testCaseSteps "simple-stack" $ testDirectory "./tests/projects/simple-stack/B.hs"
-        , testCaseSteps "simple-direct" $ testDirectory "./tests/projects/simple-direct/B.hs"
-        , testCaseSteps "simple-bios" $ testDirectory "./tests/projects/simple-bios/B.hs"
-        , testCaseSteps "multi-cabal" {- tests if both components can be loaded -}
-                      $  testDirectory "./tests/projects/multi-cabal/app/Main.hs"
-                      >> testDirectory "./tests/projects/multi-cabal/src/Lib.hs"
-        , testCaseSteps "multi-stack" {- tests if both components can be loaded -}
-                      $  testDirectory "./tests/projects/multi-stack/app/Main.hs"
-                      >> testDirectory "./tests/projects/multi-stack/src/Lib.hs"
-        ]
+      , testGroup "Loading tests"
+        $ linuxExlusiveTestCases
+        ++
+           [ testCaseSteps "simple-cabal" $ testDirectory "./tests/projects/simple-cabal/B.hs"
+           , testCaseSteps "simple-stack" $ testDirectory "./tests/projects/simple-stack/B.hs"
+           , testCaseSteps "simple-direct" $ testDirectory "./tests/projects/simple-direct/B.hs"
+           , testCaseSteps "multi-cabal" {- tests if both components can be loaded -}
+                         $  testDirectory "./tests/projects/multi-cabal/app/Main.hs"
+                         >> testDirectory "./tests/projects/multi-cabal/src/Lib.hs"
+           , testCaseSteps "multi-stack" {- tests if both components can be loaded -}
+                         $  testDirectory "./tests/projects/multi-stack/app/Main.hs"
+                         >> testDirectory "./tests/projects/multi-stack/src/Lib.hs"
+           ]
       ]
 
-
+linuxExlusiveTestCases :: [TestTree]
+linuxExlusiveTestCases
+  | isWindows = []
+  | otherwise = [ testCaseSteps "simple-bios" $ testDirectory "./tests/projects/simple-bios/B.hs" ]
```

Comment:
I'd have done `[testCaseSteps ... | not isWindows] and inlined the whole thing. Personal preference though.